lib/rules/color-function-alias-notation/README.md

```diff
@@ -0,0 +1,76 @@
+# color-function-alias-notation
+
+Specify alias notation for color-functions.
+
+<!-- prettier-ignore -->
+```css
+    a { color: rgb(0 0 0 / 0.2) }
+/**            ↑
+ *             This notation */
+```
+
+Color functions `rgb()` and `hsl()` have aliases `rgba()` and `hsla()`. Those are exactly equivalent, and [it's preferable](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb) to use the first variant without `a`.
+
+The [`fix` option](../../../docs/user-guide/options.md#fix) can automatically fix all of the problems reported by this rule.
+
+The [`message` secondary option](../../../docs/user-guide/configure.md#message) can accept the arguments of this rule.
+
+## Options
+
+`string`: `"without-alpha"|"with-alpha"`
+
+### `"without-alpha"`
```

Comment:
@jeddy3 
I am a bit too late in the discussion but I think something like a boolean rule would have been simpler.
i.e. `color-function-prefer-alias: true` or `color-function-no-alias: true` (Allowed, disallowed & required)

Currently the name of the rule seems to be targetted at the aliases themselves which are `rgba()` and `hsla()`.
i.e. not `rgb()` and `hsl()`

Reply:
I still think the name is improper/confusing but since I don't have a good alternative, I don't want to stall the publication of the next version.